website/content/docs/extending-waypoint/plugin-interfaces/platform-releaser.mdx

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+page_title: 'PlatformReleaser'
+description: |-
+  How to implement the PlatformReleaser component for a Waypoint plugin
+---
+
+# PlatformReleaser
+
+https://pkg.go.dev/github.com/hashicorp/waypoint-plugin-sdk/component#PlatformReleaser
+
+The PlatformReleaser interface is an optional interface that a Platform component
+can implement to provide default Release functionality. This only takes effect
```

Comment:
```suggestion
can implement to provide default [`Release`](/docs/extending-waypoint/plugin-interfaces/release-manager) functionality. This only takes effect
```

I think this makes sense to add font and link, but this is just a suggestion
could also link this to